.github/docker/testing/Dockerfile.testing-plugins-alma9

```diff
@@ -59,9 +64,16 @@ gpgcheck=1\n\
 gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n'\
 >> /etc/yum.repos.d/centreon-plugins.repo
 
+# Add Centreon plugins repositories
+dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/24.10/el9/centreon-24.10.repo
+dnf install -y centreon-connector-perl centreon-clib
+dnf update -y
```

Comment:
this is not recommended to do `dnf update` in a dockerfile because it can increase drastically the docker image size